docs/source/conf.py

```diff
@@ -1,9 +1,14 @@
 import os
 import sys
-from pyro import __version__
 
 import sphinx_rtd_theme
 
+if 'READTHEDOCS' not in os.environ:
+    # install pyro on rtd so it can be imported
+    os.system('pip install pyro-ppl')
```

Comment:
This will set `pyro.__version__` to whatever is on `PyPi`, not what is in the local file system. Perhaps you should instead `pip install .` or similar?

Reply:
sorry I still don't understand. Is this fix intended for local development in environments where pyro has not been installed? Or is it for travis to check that docs can be built?

Reply:
Don't query pypi for the Pyro version when building docs on rtd. This would break every time we update the version, since rtd docs are created on git push, whereas pypi version is updated on `python setup.py sdist upload`.

Just hard-code the version.

Reply:
> Just hard-code the version.

as in the snippet above?

and just to be clear, rtd doesnt actually *use* the version, so even if the pypi is out of date, it wouldnt matter

Reply:
If rtd doesn't use the version then set `__version__ = None` or something?

Reply:
Or set `__version__ = 'ignored'`